app/models/pglogical_subscription.rb

```diff
@@ -108,10 +108,16 @@ def validate(new_connection_params = {})
   end
 
   def backlog
-    connection.xlog_location_diff(remote_node_lsn, remote_replication_lsn)
-  rescue PG::Error => e
-    _log.error(e.message)
-    nil
+    if status != "replicating"
+      _log.error("Please check that database `#{dbname}` is running on host `#{host}` and accepting TCP/IP connections on port #{port}")
```

Comment:
Curious why this log message is different from the one on the master PR?

Reply:
no reason, I had 2 environment 5.10 and 5.11 and always was mistake what i am testing/looking at the log, I will change it to be the same ...